### ISSUE TITLE
Missing background on sticky scroll lines

### ISSUE THREAD

**Issue by erlete** [KatsuteDev/Background#277]
### Operating System

Windows

### VSCode Version

1.85.1

### Extension Version

2.9.1

### Issue

Sticky scroll lines from core VSCode Settings are not affected by the background settings, but rather default to the applied theme's color.

![image](https://github.com/KatsuteDev/Background/assets/76848729/b9264e9c-3a74-4ea4-9496-cb8134a287bd)

I have set multiple images, tweaked plenty of background customization settings, but the issue remains unaffected.

### Settings

```json
"background.windowBackgrounds": [
    <hidden for privacy reasons>
],
"background.backgroundOpacity": [
    0.8,
    0.9,
    0.9,
    0.9
],
"background.backgroundSize": [
    "Cover",
    "Cover",
    "Cover",
    "Cover"
],
"background.backgroundBlur": [
    "0.1",
    "0",
    "0",
    "0"
],
"background.backgroundSizeValue": [
    "100%",
    "100%",
    "100%",
    "100%"
],
"background.smoothImageRendering": true,
"background.renderContentAboveBackground": true,
"background.useWindowOptionsForAllBackgrounds": true
```

### How to replicate

1. Install extension
2. Add background images
3. Install & reload
4. Enable `editor.stickyScroll.enabled` on VSCode Settings
5. Navigate to a file where sticky scroll can be seen
6. ***Behold!***

**Comment by Katsute** [comment]
This can be fixed by setting the `background.renderContentAboveBackground` option to `false`.

**Comment by erlete** [comment]
Closed due to same root cause as in #276.
